test/logs/utils/log.ts

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs';
+
+export interface UnsupportedTypeLog {
+  header: string;
+  created: string;
+  usedBy: string;
+}
+
+export const getLogsByCreateMockFileName: (
+  fileName: string
+) => UnsupportedTypeLog[] = (fileName: string) => {
+  const logErrorFile: string = fs.readFileSync('./tsAutoMock.log', 'utf-8');
+  const logs: UnsupportedTypeLog[] = [];
+  const lines: string[] = logErrorFile
+    .split('\n')
+    .filter((line: string) => !!line);
+
+  for (let i: number = 0; i < lines.length; i += 3) {
+    logs.push({
+      header: lines[i],
+      created: lines[i + 1],
```

Comment:
This is extremely opinionated in a way that if we have any other way to write warnings (for example a one line warning in style "you can't use that while using random feature") we have to update this function... I think I'm ok with it, I just wanted to point it out

Reply:
I agree. I would say that once we need it we can improve the utility to get the logs :)

Reply:
Another way to do it would be to parse the warnings based on the service name. For example ... divide the logs starting from WARNING - {serviceName}, nothing to difficult.